test/Main.hs

```diff
@@ -60,5 +62,56 @@ parseTemplate
     parseJSON :: Text -> Either String Paths.Template
     parseJSON = JSON.parseEither JSON.parseJSON . JSON.String
 
+parseResponses :: TestTree
+  = testGroup "OpenAPI.Path.Responses parsing"
+  $ accepted <> rejected
+  where
+    accepted :: [TestTree]
+    accepted = mkAccepted <$>
+      [ (JSON.object [], Paths.Responses { default' = empty, patterns = Map.empty })
+      , (responsesJSON, responses)
+      ]
+
+    responses :: Paths.Responses
+    responses = Paths.Responses {default' = empty, ..}
+      where
+        patterns = Map.fromList
+          [ (Paths.ResponseStatusExact HTTP.status200, response) ]
+
+        response = Paths.Response
+          { description = empty
+          , headers     = empty
+          , content     = Map.empty
+          }
+
+    responsesJSON :: JSON.Value
+    responsesJSON =
+      JSON.object
+        [ ("200", JSON.object [("content", JSON.object [])])]
```

Comment:
The `]` at the end of the line is missing a leading space; which will make it consistent with the other code in this commit/file.